internal/impl/tmpl/devcontainerDockerfile.tmpl

```diff
@@ -2,7 +2,7 @@ FROM debian:stable-slim
 
 # Step 1: Installing dependencies
 RUN apt-get update
-RUN apt-get -y install bash binutils git xz-utils wget sudo
+RUN apt-get -y install bash binutils git{{if .IsDevcontainer}} gnupg2{{- end}} xz-utils wget sudo
```

Comment:
Thanks! Makes sense, since devcontainer Dockerfile is usually used in VSCode context which signed commits are not that uncommon.